build_chalice_config.sh

```diff
@@ -34,4 +34,15 @@ else
            .$stage.lambda_functions = {}" > "$deployed_json"
 fi
 
+cat "$config_json" | jq ".stages.$stage.api_gateway_stage=env.stage" | sponge "$config_json"
+export DEPLOY_ORIGIN="$(whoami)-$(hostname)-$(git describe --tags --always)-$(date -u +'%Y-%m-%d-%H-%M-%S').deploy"
+cat "$config_json" | jq .stages.$stage.tags.FUS_DEPLOY_ORIGIN=env.DEPLOY_ORIGIN | sponge "$config_json"
+env_json=$(aws ssm get-parameter --name /${FUS_PARAMETER_STORE}/${FUS_DEPLOYMENT_STAGE}/environment | jq -r .Parameter.Value)
```

Comment:
The purpose of the SSM store in the DSS is to decouple environment variables from deployment. For instance when you want to update runtime code _without_ updating Lambda configuration.

Does this use case exist in fusillade? If not, it may be preferable to not use the SSM store.

Reply:
I see what you mean. Not immediately, but I think we will want this feature for modifying debug, and logging parameters.